lib/mediaType.js

```diff
@@ -120,15 +120,39 @@ function specify(type, spec, index) {
 
 }
 
-function preferredMediaTypes(accept, provided) {
+/**
+ * Returns a list of the acceptable media ranges in preferred order.
+ *
+ * @param {string} accept The raw contents of the `Accept` header
+ * @param {string[]} provided The media ranges the server can produce
+ *     for the resource in question.
+ * @param {Object} options Configuration options
+ * @param {boolean} options.detailed When `provided` is not defined, such
```

Comment:
Sure, we can make it work without `provided` too! I've updated the PR to do that.